src/gwt/src/org/rstudio/core/client/widget/ProgressIndicatorDelay.java

```diff
@@ -50,6 +55,7 @@ public void run()
                if (showing_)
                {
                   progressIndicator_.onProgress(status);
+                  onShowProgress.execute();;
```

Comment:
Before you invest any time in the socket based approach we should make sure
that you are solving a real performance problem. All of htmlwidgets is
based on writing and reading many large temp files and it all ends up
happening in "user time" (less than 50ms). To me we'd want the socket if we
got into the dozens of megabytes of data but not much before that.

On Mon, Feb 15, 2016 at 11:13 AM, Javier Luraschi <notifications@github.com>
wrote:

> In src/gwt/src/org/rstudio/core/client/widget/ProgressIndicatorDelay.java
> <https://github.com/rstudio/rstudio/pull/587#discussion_r52929216>:
>
> > @@ -50,6 +55,7 @@ public void run()
> >                 if (showing_)
> >                 {
> >                    progressIndicator_.onProgress(status);
> > +                  onShowProgress.execute();;
>
> Thanks, updated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rstudio/pull/587/files#r52929216>.
>